src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/GoalEntityMapperTest.kt

```diff
@@ -27,8 +28,8 @@ class GoalEntityMapperTest {
   @InjectMocks
   private lateinit var mapper: GoalEntityMapperImpl
 
-  @Mock
-  private lateinit var stepMapper: StepEntityMapper
+  @Spy
+  private val stepMapper = StepEntityMapperImpl()
```

Comment:
😱 😉

Reply:
Ah - I might be able to put this back to a mock now, now that the mapper is much simplified. Let me see if I can change it back (I hate the use of spy as well!)

Reply:
Done; nice; thank you